media/js/teach/smarton.js

```diff
@@ -0,0 +1,303 @@
+    }
+
+    if (window.isFirefox()) {
+        $body.addClass('is-firefox');
+    } else {
+        $body.addClass('not-firefox');
+    }
+
+    // Sticky navigation
+    var fixed = false;
+    var didScroll = false;
+
+    $window.scroll(function() {
+        didScroll = true;
+    });
+
+    if ($header.length > 0) {
+        $(document).ready(function() {
+            var scrollTop = $window.scrollTop();
+            if (scrollTop >= headTop.top) {
+                didScroll = true;
+            }
+        });
+    }
+
+    function adjustScrollbar() {
+        if (didScroll) {
+            didScroll = false;
+            var scrollTop = $window.scrollTop();
+            if ($header.length > 0) {
+                if( scrollTop >= headTop.top ) {
+                    if(!fixed) {
+                        fixed = true;
+                        $header.addClass('fixed');
+                        $firstStep.addClass('current');
+                        // recalculate height after sticking
+                        headHeight = $header.height();
+                        // then pad the first section so it doesn't jump as far
+                        $firstSection.css({ paddingTop: headHeight + 40 });
+                    }
+                } else {
+                    if(fixed) {
+                        fixed = false;
+                        $header.removeClass('fixed');
+                        $firstStep.removeClass('current');
+                        $firstSection.removeAttr('style');
+                        $header.find('.nav-steps li').removeClass();
+                    }
+                }
+            }
+        }
+    }
+
+    // Check for an adjusted scrollbar every 100ms.
+    setInterval(adjustScrollbar, 100);
+
+    // Scroll smoothly to the linked section
+    $document.on('click', '.nav-steps a[href^="#"]', function(e) {
+        e.preventDefault();
+        // Extract the target element's ID from the link's href.
+        var elem = $(this).attr('href').replace( /.*?(#.*)/g, "$1" );
+        $('html, body').animate({
+            scrollTop: $(elem).offset().top - 83
+        }, 1000, function() {
+            $(elem).attr('tabindex','100').focus().removeAttr('tabindex');
+        });
+    });
+
+    var $footer = $('#footer-cta');
+    var footHeight = $footer.height();
+    var tallMode = false;
+    var queryHeight = matchMedia('(min-height: 650px)');
+    var wideMode = false;
+    var queryWidth = matchMedia('(min-width: 768px)');
+
+    // Check window height
+    if (queryHeight.matches) {
+        tallMode = true;
+    } else {
+        tallMode = false;
+    }
+
+    queryHeight.addListener(function(mq) {
+        if (mq.matches) {
+            tallMode = true;
+        } else {
+            tallMode = false;
+        }
+    });
+
+    // Check window width
+    if (queryWidth.matches) {
+        wideMode = true;
+    } else {
+        wideMode = false;
+    }
+
+    queryWidth.addListener(function(mq) {
```

Comment:
This is throwing an error in IE9 - need to include the polyfill in a conditional comment [like here](https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/templates/mozorg/about/manifesto.html#L337-L339)